purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/RedeemWebResultListener.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.interfaces
+
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.PurchasesError
+
+/**
+ * Interface to handle the redemption of a RevenueCat Web purchase.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+fun interface RedeemWebResultListener {
```

Comment:
Naming suggestion: `RedeemWebPurchaseListener`. (Mostly focused on getting the word "purchase" in there.)